resources/assets/components/Campaign/Campaign.js

```diff
@@ -30,10 +30,9 @@ const Campaign = props => (
       <LegacyAdminDashboardContainer>
         <LegacyCampaignDashboardContainer />
       </LegacyAdminDashboardContainer>
-
       <NotificationContainer />
 
-      {props.isAuthenticated && featureFlag('nps_survey') ? (
+      {props.isAuthenticated && !featureFlag('sitewide_nps_survey') ? (
```

Comment:
This is not a bit deal as I don't think we intend to turn off the main `npm_survey` feature flag, but perhaps we should leave it in still?
```suggestion
      {props.isAuthenticated && featureFlag('nps_survey') &&  !featureFlag('sitewide_nps_survey') ? (
```

Reply:
ooohhhhh, thanks!